Cargo.toml

```diff
@@ -257,16 +257,16 @@ tree-sitter-ocaml = { git = "https://github.com/tree-sitter/tree-sitter-ocaml",
 tree-sitter-php = "0.21.1"
 tree-sitter-prisma-io = { git = "https://github.com/victorhqc/tree-sitter-prisma" }
 tree-sitter-proto = { git = "https://github.com/rewinfrey/tree-sitter-proto", rev = "36d54f288aee112f13a67b550ad32634d0c2cb52" }
-tree-sitter-purescript = { git = "https://github.com/ivanmoreau/tree-sitter-purescript", rev = "a37140f0c7034977b90faa73c94fcb8a5e45ed08" }
+tree-sitter-purescript = { git = "https://github.com/kazatsuyu/tree-sitter-purescript", rev = "e21296ba922de592bd49365f6ada4deef6bbe9a5" }
```

Comment:
I would not like to maintain this repository for long term. However, tree-sitter will move to extensions, so if it is ok to keep this repository for the short term, I will remove the draft mark.
Since this repository is a fork of a fork of a fork of a fork, we should be able to use one of the upstreams in the future.

Reply:
Since `tree-sitter-purescript` is already ignored on linux, I decided to remove it as a dependency once rather than blocking work for it.